core/src/main/java/org/apache/accumulo/core/iterators/user/ColumnAgeOffFilter.java

```diff
@@ -43,6 +43,9 @@ public TTLSet(Map<String,String> objectStrings) {
       for (Entry<String,String> entry : objectStrings.entrySet()) {
         String column = entry.getKey();
         String ttl = entry.getValue();
+        // skip the negate option, it will cause an exception to be thrown
+        if (column.equalsIgnoreCase(NEGATE) && (ttl.equalsIgnoreCase("true") || ttl.equalsIgnoreCase("false")))
```

Comment:
Nitpick: the column check could change to `column.equals(NEGATE)` since Filter performs a case-sensitive checking.  But this is fine as is.  Thanks @milleruntime!